b2luigi/batch/processes/gbasf2.py

```diff
@@ -585,12 +612,33 @@ def _download_dataset(self):
             tmp_output_dir_path = f"{output_dir_path}.partial"
             os.makedirs(tmp_output_dir_path, exist_ok=True)
 
-            ds_get_command = shlex.split(f"gb2_ds_get --force {dataset_query_string}")
-            print("Downloading dataset with command ", " ".join(ds_get_command))
+            # Need a file to repeat download for FAILED ones only
+            monitoring_failed_downloads_file = os.path.join(tmp_output_dir_path, "failed_files.txt")
+
+            # In case of first download, this file does not exist
+            if not os.path.exists(monitoring_failed_downloads_file):
```

Comment:
- [ ] use the more specific `isfile`
```suggestion
            if not os.path.isfile(monitoring_failed_downloads_file):
```

Reply:
fixed